samples/dotnet-isolated/confluent/KafkaTriggerManyWithHeaders.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Http;
+using Newtonsoft.Json.Linq;
+
+
+namespace Confluent
+{
+    public class KafkaTriggerManyWithHeaders
```

Comment:
Is there any difference b/w code of KafkaTriggerManyWithHeaders and  KafkaTriggerMany ?

Reply:
Yes, there's a difference between both functions. KafkaTriggerMany is sample for kafka trigger with messages in batch mode while KafkaTriggerManyWithHeaders is sample for kafka trigger with messages and headers.